appveyor.yml

```diff
@@ -1,64 +1,61 @@
 build: false
 
-init:
-  - SET PATH=%PATH%;C:\Program Files\OpenSSL;c:\tools\php;c:\cygwin\bin
-
-  # uncomment to allow remote desktop connection
-  #- ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
-on_finish:
-  # uncomment to show remote desktop connection details and block build
-  #- ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+environment:
+ matrix:
+    - PHP_VERSION: 7.0.27 # see http://windows.php.net/downloads/releases/ for available versions
+      JACKRABBIT_VERSION: '2.12.0'
+      SYMFONY__PHPCR__TRANSPORT: 'jackrabbit'
```

Comment:
seems like before jackrabbit was installed but never set the transport to jackrabbit?